.github/workflows/build_app.yml

```diff
@@ -58,7 +58,7 @@ jobs:
       - run: mkdir -p ./spec/decidim_dummy_app/tmp/screenshots
         name: Create the screenshots folder
         shell: "bash"
-      - run: RAILS_ENV=test bundle exec rails assets:precompile
+      - run: RAILS_ENV=test bundle exec rails shakapacker:compile
```

Comment:
Even though in a normal app we can get away with `assets:precompile`, in the CI it seems like it only works with `shakapacker:compile` 🤷🏽‍♂️